### PR TITLE
Bugfix reset inputs on failed signup/login

### LINKED ISSUE #104: Reset user inputs on failed signup/login
If the signup/login process fails, we should reset the inputs so the user can try again
Depends on #13 & #14

### PR DESCRIPTION
**Pull request by max-gurr**
This was implemented for signup, but must have gotten removed at some point. It's just resetting the user inputs if either the signup fails (in the case where all inputs were supplied, so it's an authentication issue), or when the login details weren't correct.

### CODE REVIEW

**Review by jzen349 — APPROVED** [review]
LGTM ! :)

**Review by franc-hoon — APPROVED** [review]
inconsistency in using ' and " but other than that LGTM

**Review by lucas2005gao — APPROVED** [review]
Nice one!!

File: src/pages/loginView.js [hunk 1]
```diff
@@ -31,13 +31,16 @@ export default function LoginView() {
...
     if (validLogin) {
-      // If successful, can go to /home
+      // If authentication is successful, can go to /home
       if (await signIn(processedEmail, processedPassword)) {
         history.push("/home");
       } else {
+        // If authentication is unsuccessful, notify user and reset inputs
         alert("Invalid login");
+
+        setEmail("");
```
Comment (franc-hoon) [inline src/pages/loginView.js]: Now that I think about it, most websites don't reset the email field when it's an invalid login

**Review by franc-hoon — CHANGES_REQUESTED** [review]

File: src/pages/loginView.js (hunk 1 above)
Comment (max-gurr) [reply src/pages/loginView.js]: Good thinking!

**Review by franc-hoon — APPROVED** [review]
LGTM

**Pull request merged**